metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/StringFunctions.java

```diff
@@ -343,4 +343,89 @@ public Object apply(List<Object> args) {
       return String.format(format, formatArgs);
     }
   }
+
+  @Stellar( name="CHOP"
```

Comment:
If we are going to do URL/URI work, we should create functions off of the URI / URL classes and use their specific parsing for uri named component parts